lib/exe.js

```diff
@@ -134,6 +134,12 @@ async function main () {
     }
   }
 
+  if (process.pid) {
+    log.i('This process is your PID ' + process.pid)
+    console.log('#PID#:' + process.pid)
+    console.log('#PID-READY#')
```

Comment:
we may want to standardize some launch feedback mechanism at some point so that (a) we can publish it all at once, waiting on the n3h side to simplify the logic in rust, and (b) potentially use json giving us more expressiveness, like:

```
#-n3h-ipc-launch-feedback-marker-start-#
{
  "pid": 3664,
  "ipcBinding": "wss://127.0.0.1:3333/",
  "p2pBindings": [
    "wss://1.1.1.1:55888/"
  ]
}
#-n3h-ipc-launch-feedback-marker-end-#
```

just a thought for the future

Reply:
ok, i'll keep that in mind. 
Not too sure about it since different modes might give different 'feedback info'. I changed mockmode to output the p2p-binding for consistency but its a bit hackish since it doesnt really have one.
Also, not sure if we are going to keep the `pid` if it was just for my bat file hack.